packages/terra-switch/src/Switch.jsx

```diff
@@ -0,0 +1,136 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import * as KeyCode from 'keycode-js';
+import styles from './Switch.module.scss';
+
+const cx = classNames.bind(styles);
+
+// Translations Required
+const SWITCH_STATE = Object.freeze({
+  ON: 'On',
+  OFF: 'Off',
+});
+
+const propTypes = {
+  /**
+   * Id of the Switch button
```

Comment:
Nit: Please add punctuation (period) to be consistent with the other comments.

Reply:
updated : [here](https://github.com/cerner/terra-core/pull/2846/commits/a30ab9bf52d60ad2b5b83dbe86defbb2a580b981#diff-ce03daf8e4362de3a438b2f322dedf6f)